### PR TITLE
CRYPTO80:Fix the PMD error

### PR DESCRIPTION
**Pull request by JkSelf**
fix the pmd error

### CODE REVIEW

File: src/main/java/org/apache/commons/crypto/utils/OSInfo.java [hunk 1]
```diff
@@ -168,9 +168,9 @@ public static String getArchName() {
                 if (exitCode == 0) {
                     return "armhf";
                 }
-            } catch (IOException e) {
-                // ignored: fall back to "arm" arch (soft-float ABI)
-            } catch (InterruptedException e) {
+            } catch (IOException e) { //NOPMD
+            // ignored: fall back to "arm" arch (soft-float ABI)
```
Comment (sundapeng) [inline src/main/java/org/apache/commons/crypto/utils/OSInfo.java]: Add 4 spaces here

File: src/main/java/org/apache/commons/crypto/utils/Utils.java [hunk 2]
```diff
@@ -126,8 +114,9 @@ public static void freeDirectBuffer(ByteBuffer buffer) {
                     return;
                 }
             }
-        } catch (ReflectiveOperationException e) {
-            // Ignore the Reflection exception.
+        } catch (ReflectiveOperationException e) { // NOPMD
+             // Ignore the Reflection exception.
```
Comment (sundapeng) [inline src/main/java/org/apache/commons/crypto/utils/Utils.java]: reduce 1 space

File: src/main/java/org/apache/commons/crypto/utils/Utils.java [hunk 3]
```diff
@@ -159,13 +148,13 @@ public static int getBufferSize(Properties props) {
...
      */
     public static String getCipherClassString(Properties props) {
-        final String configName = COMMONS_CRYPTO_CIPHER_CLASSES_KEY;
+        final String configName = ConfigurationKeys.COMMONS_CRYPTO_CIPHER_CLASSES_KEY;
         String cipherClassString = props.getProperty(configName) != null ? props
-                .getProperty(configName, COMMONS_CRYPTO_CIPHER_CLASSES_DEFAULT)
+                .getProperty(configName, ConfigurationKeys.COMMONS_CRYPTO_CIPHER_CLASSES_DEFAULT)
                 : System.getProperty(configName,
-                        COMMONS_CRYPTO_CIPHER_CLASSES_DEFAULT);
+            ConfigurationKeys.COMMONS_CRYPTO_CIPHER_CLASSES_DEFAULT);
```
Comment (sundapeng) [inline src/main/java/org/apache/commons/crypto/utils/Utils.java]: need to add more space

File: src/main/java/org/apache/commons/crypto/utils/Utils.java [hunk 4]
```diff
@@ -192,11 +181,11 @@ public static String getJCEProvider(Properties props) {
      */
     public static String getRandomDevPath(Properties props) {
         String devPath = props
-                .getProperty(COMMONS_CRYPTO_SECURE_RANDOM_DEVICE_FILE_PATH_KEY);
+                .getProperty(ConfigurationKeys.COMMONS_CRYPTO_SECURE_RANDOM_DEVICE_FILE_PATH_KEY);
         if (devPath == null) {
             devPath = System.getProperty(
-                    COMMONS_CRYPTO_SECURE_RANDOM_DEVICE_FILE_PATH_KEY,
-                    COMMONS_CRYPTO_SECURE_RANDOM_DEVICE_FILE_PATH_DEFAULT);
+                ConfigurationKeys.COMMONS_CRYPTO_SECURE_RANDOM_DEVICE_FILE_PATH_KEY,
```
Comment (sundapeng) [inline src/main/java/org/apache/commons/crypto/utils/Utils.java]: 4 more space

File: src/main/java/org/apache/commons/crypto/utils/Utils.java [hunk 5]
```diff
@@ -192,11 +181,11 @@ public static String getJCEProvider(Properties props) {
...
+                ConfigurationKeys.COMMONS_CRYPTO_SECURE_RANDOM_DEVICE_FILE_PATH_DEFAULT);
```
Comment (sundapeng) [inline src/main/java/org/apache/commons/crypto/utils/Utils.java]: 4 more space

File: src/main/java/org/apache/commons/crypto/utils/Utils.java [hunk 6]
```diff
@@ -178,9 +167,9 @@ public static String getCipherClassString(Properties props) {
      * @return the jce provider based on the props.
      */
     public static String getJCEProvider(Properties props) {
-        return props.getProperty(COMMONS_CRYPTO_CIPHER_JCE_PROVIDER_KEY) != null ? props
-                .getProperty(COMMONS_CRYPTO_CIPHER_JCE_PROVIDER_KEY) : System
-                .getProperty(COMMONS_CRYPTO_CIPHER_JCE_PROVIDER_KEY);
+        return props.getProperty(ConfigurationKeys.COMMONS_CRYPTO_CIPHER_JCE_PROVIDER_KEY) != null ? props
```
Comment (sundapeng) [inline src/main/java/org/apache/commons/crypto/utils/Utils.java]: please check if the line is too long.

File: src/main/java/org/apache/commons/crypto/utils/Utils.java [hunk 7]
```diff
@@ -31,17 +31,7 @@
...
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_CIPHER_JCE_PROVIDER_KEY;
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_LIB_NAME_KEY;
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_LIB_PATH_KEY;
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_LIB_TEMPDIR_KEY;
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_SECURE_RANDOM_DEVICE_FILE_PATH_DEFAULT;
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_SECURE_RANDOM_DEVICE_FILE_PATH_KEY;
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_STREAM_BUFFER_SIZE_DEFAULT;
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_STREAM_BUFFER_SIZE_KEY;
-import static org.apache.commons.crypto.conf.ConfigurationKeys.COMMONS_CRYPTO_SYSTEM_PROPERTIES_FILE;
+
```
Comment (sundapeng) [inline src/main/java/org/apache/commons/crypto/utils/Utils.java]: remove this line

**Comment by sundapeng** [comment]
Thank @JkSelf for your contribution!

**Comment by JkSelf** [comment]
Thanks @sundapeng  comment. I have fixed.

**Comment by sundapeng** [comment]
+1, Thank @JkSelf for your contribution.

**Comment by sundapeng** [comment]
Committed, thank @JkSelf

**Pull request closed without merging**